workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/ProfileDropdown.tsx

```diff
@@ -61,6 +72,20 @@ export const ProfileDropdown = ({ layout }: ProfileDropdownProps) => {
     }
   }, []);
 
+  useEffect(() => {
+    const fetchUserEntity = async () => {
+      let userProfile;
+      if (backstageIdentity?.userEntityRef) {
+        userProfile = (await catalogApi.getEntityByRef(
```

Comment:
Do you think we need a try catch block here for the entity fetching?

Reply:
Added!